### ISSUE TITLE
`false` values must be quoted

### ISSUE THREAD

**Issue by johnpash** [checkout/checkout-sdk-ruby#4]
I ran into a small problem using the api to construct a non-capture request. Here is the code:
```ruby
source = CheckoutSdk::PaymentRequestSource.new
source.capture = false
```
 
However, the [delete_blank](https://github.com/checkout/checkout-sdk-ruby/blob/master/lib/checkout_sdk/api_resource.rb#L71) method was resolving `false` to "blank" causing the key/value pair to be dropped.

The fix is to simply quote the value:
```ruby
source.capture = 'false'
```

The reason I got confused was due to [the api documentation](https://api-reference.checkout.com/#tag/Payments) showing json and c# code with a bare `false` keyword.

Thinking about it more, I believe the code to be correct but perhaps the documentation for this gem could have an example or warning that "false" needs to be treated in this way. I'm happy to open a PR but wanted to check in case the code is in fact wrong. What do you think?

**Comment by nicolas-maalouf-cko** [comment]
Hi @johnpash I agree this can be confusing 

Would your PR involve updating the delete_blank function to ignore booleans? or you were thinking of updating the docs?

Thanks

**Comment by johnpash** [comment]
I had a chat with a colleague about this and we thought it made more sense to support the keyword `false`  and send that through to the API. I've opened a PR here #5

**Comment by nicolas-maalouf-cko** [comment]
Fixed in [v0.2.1](https://github.com/checkout/checkout-sdk-ruby/releases/tag/v0.2.1)

Thanks for your help
